File: mobile-client/ios/Contract/Shipment/Detail/ShipmentDetailPayload.swift
import Foundation

// Marketing America Corp. Oleksandr Tishchenko
struct ShipmentDetailPayload {
    let shipmentId: String
    let shipmentNumber: String
    let orderId: String
    let carrierCode: String
    let trackingNumber: String
    let currentStatusCode: String
    let latestEventLabel: String
    let destinationLabel: String
}

File: mobile-client/android/Contract/Shipment/Detail/ShipmentDetailPayload.kt
package app.mobiling.client.client.contract.shipment.detail

/**
 * Marketing America Corp. Oleksandr Tishchenko
 */
data class ShipmentDetailPayload(
    val shipmentId: String,
    val shipmentNumber: String,
    val orderId: String,
    val carrierCode: String,
    val trackingNumber: String,
    val currentStatusCode: String,
    val latestEventLabel: String,
    val destinationLabel: String,
)
